src/blockchain/bitcoin/block/header/Header.hpp

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2010-2022 The Open-Transactions developers
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include <boost/endian/buffers.hpp>
+#include <boost/endian/conversion.hpp>
+#include <array>
+#include <cstdint>
+#include <memory>
+
+#include "blockchain/block/header/Header.hpp"
+#include "blockchain/block/header/Imp.hpp"
+#include "internal/blockchain/bitcoin/block/Header.hpp"
+#include "internal/blockchain/block/Block.hpp"
+#include "opentxs/blockchain/BlockchainType.hpp"
+#include "opentxs/blockchain/Types.hpp"
+#include "opentxs/blockchain/bitcoin/NumericHash.hpp"
+#include "opentxs/blockchain/bitcoin/Work.hpp"
```

Comment:
Redundant. Please revise all other includes as seems that a lot of them are not used.